test/test_loaders.py

```diff
@@ -1,6 +1,7 @@
 import json
 
 import pytest
+from yarl import URL
```

Comment:
I believe we need to add `yarl` to direct dependencies via `poetry add yarl`

Reply:
Done